config/locales/en.yml

```diff
@@ -80,6 +80,7 @@ en:
         other_phone: Other phone
         patient_contribution: Patient contribution
         pledge_generated_at: Pledge generated at
+        pledge_generated_by: Pledge generated by
```

Comment:
Sorry, looks like github ate my comment here! Basically the rails-ism is that in database this is `pledge_generated_by_id` -- `pledge_generated_by` is a rails shorthand that signals that you want the User object, which we almost always do, but not here because we're dealing with changes to the id value. See below -- note that the `translation missing` error refers 

So basically, turn this and the spanish one into `pledge_generated_by_id` and you're all set, I think.

<img width="821" alt="image" src="https://github.com/DARIAEngineering/dcaf_case_management/assets/3866868/5f277861-e2cd-4b14-8083-b169c69b42f1">

Reply:
done - lmk if there's any other changes you'd like to see!